pkg/wallet/wallet.go

```diff
@@ -28,7 +28,7 @@ type Wallet struct {
 
 	// Extra metadata can be used for storing arbitrary data.
 	// This field can be empty.
-	Extra interface{} `json:"extra"`
+	Extra Extra `json:"extra"`
```

Comment:
I'm not sure about this. It obviously is more effective storing these values, but at the same time it makes us update the wallet for every NEP5 token and may affect interoperability.

Reply:
It is fully compliant with NEP-6 standard.